workflow/rocoto/gfs_tasks.py

```diff
@@ -796,6 +796,66 @@ def postsnd(self):
 
         return task
 
+    def fbwinds(self):
+
+        def _get_fbwindsgroups(cdump, config):
+
+            fhmin = config['FHMIN']
+            fhmax = config['FHMAX']
+            fhout = config['FHOUT']
+
+            # Get a list of all forecast hours
+            fhrs = []
+            if cdump in ['gdas']:
+                fhrs = range(fhmin, fhmax + fhout, fhout)
+            elif cdump in ['gfs']:
+                fhmax = np.max(
+                    [config['FHMAX_GFS_00'], config['FHMAX_GFS_06'], config['FHMAX_GFS_12'], config['FHMAX_GFS_18']])
+                fhout = config['FHOUT_GFS']
+                fhmax_hf = config['FHMAX_HF_GFS']
+                fhout_hf = config['FHOUT_HF_GFS']
+                if fhmax > 240:
+                    fhmax = 240
```

Comment:
why is this being hard-wired here? 
Is it because `FHMAX_FBWIND` is `240`?  If so, we should define it in `config.base` or `config.fbwind`.  Could hold it for another PR.  If so, please add a `TODO`

Reply:
None of this is necessary at all. `fbwinds` should be a single job, not a metatask. And it only cares about post being completed for `06`, `12`, and `24`:

https://github.com/NOAA-EMC/global-workflow/blob/62d8e99e77ffe44784f4c61c3f2296c6d1a72413/scripts/exgfs_atmos_fbwind.sh#L39C1-L51C5

Reply:
@WalterKolczynski-NOAA So how should this be rewritten? Can you provide an example or suggestion?